src/utils/index.js

```diff
@@ -0,0 +1,13 @@
+const bugsnag = require('@bugsnag/js');
+const { BUGSNAG_API_KEY } = require('../../config/config');
+
+const bugsnagClient = bugsnag(BUGSNAG_API_KEY);
+
+function catchUnexpectedError(error) {
```

Comment:
You can call it `logError(error, options)`. 
To do the same thing than `this.error(...)`, you can include `const Errors = require("@oclif/errors");` and call `Errors.error(input, options);`.

So then you can decide to exit or not by doing:
```
logError('Error message', { exit: 1 });
```